mne/viz/_3d.py

```diff
@@ -1947,3 +1948,133 @@ def _dipole_changed(event, params):
+                     title=None, threshold='auto', annotate=True,
+                     black_bg=False, cmap=None, alpha=0.7, vmin=None,
+                     vmax=None, plot_abs=True, symmetric_cbar="auto"):
+    """Plot stc on glass brain.
+
+    Parameters
+    ----------
+    stc : instance of SourceEstimate
+        Data to plot.
+    subject : str
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+        The default is None.
+    fwd : instance of SourceSpace
+        The forward solution. Only needed for VolumeSourceEstimate.
+    time : float
+        The time instance to plot in seconds. Defaults to 0.
+    display_mode : string
+        Choose the direction of the cuts: 'x' - sagittal, 'y' - coronal,
+        'z' - axial, 'l' - sagittal left hemisphere only,
+        'r' - sagittal right hemisphere only, 'ortho' - three cuts are
+        performed in orthogonal directions. Possible values are: 'ortho',
+        'x', 'y', 'z', 'xz', 'yx', 'yz', 'l', 'r', 'lr', 'lzr', 'lyr',
+        'lzry', 'lyrz'. Defaults to 'ortho'.
+    colorbar : bool
+        If True, display a colorbar on the right of the plots. Defaults to
+        False.
+    axes : matplotlib axes or 4 tuple of float: (xmin, ymin, width, height)
+        The axes, or the coordinates, in matplotlib figure space,
+        of the axes used to display the plot. If None, the complete
+        figure is used.
+    title : string
+        The title displayed on the figure.
+    threshold : float | None | 'auto'
+        The value used to threshold the image: values below the threshold (in
+        absolute value) are plotted as transparent. If None, the image is not
+        hresholded. If float, If auto, the threshold is determined magically by
+        analysis of the image.
+    annotate : bool
+        If True, positions and left/right annotation are added to the plot.
+    black_bg : bool
+        If True, the background of the image is set to be black. If
+        you wish to save figures with a black background, you
+        will need to pass "facecolor='k', edgecolor='k'"
+        to matplotlib.pyplot.savefig.
+    cmap : matplotlib colormap
+        The colormap for specified image
+    alpha : float
+        Alpha transparency for the brain schematics.
+    vmax : float
+        Upper bound for plotting, passed to matplotlib.pyplot.imshow.
+    plot_abs : bool
+        If True (default), maximum intensity projection of the absolute value
+        will be used (rendering positive and negative values in the same
+        manner). If False, the sign of the maximum intensity will be
+        represented with different colors. See
+        http://nilearn.github.io/auto_examples/01_plotting/plot_demo_glass_brain_extensive.html
```

Comment:
use ReST to call this `nilearn doc` or something, no need to display a long URL in our docs when rendered online